index.js

```diff
@@ -32,6 +32,15 @@ module.exports = () => {
       incoming_webhook_channel: req.headers['bb-incomingwebhookchannel']
     })
 
+    // Add custom config
+    req.slapp.meta.config = Object.keys(req.headers)
```

Comment:
Since this is per team config do you think it should be named as such? I'm 50/50 on it.